python/nav/web/templates/seeddb/_seeddb_netbox_form_content.html

```diff
@@ -1,4 +1,55 @@
 {% extends 'custom_crispy_templates/_form_content.html' %}
 
 {% block form_fields %}
+  <div class="row">
+    <div class="columns {{ columns_css_class|default:'large-12' }}">
+      <fieldset>
+        <legend>Inventory</legend>
+        {% include 'foundation-5/field.html' with field=form.ip %}
+        <div id="verify-address-feedback"></div>
+        {% include 'foundation-5/field.html' with field=form.room %}
+        {% include 'foundation-5/field.html' with field=form.category %}
+        {% include 'foundation-5/field.html' with field=form.organization %}
```

Comment:
This gets tedious very quickly.  Especially with the repeated template name (which is very connected to foundation 5).  Maybe we should add our own form field renderer tag so we don't need to change all these inclusions the day we change the template path (or change something else about the template) ?

If so, not a task for *this* PR, but we should definitely look into it..